dashbuilder/dashbuilder-backend/dashbuilder-services/src/main/java/org/dashbuilder/transfer/DataTransferServicesImpl.java

```diff
@@ -108,22 +114,37 @@ public DataTransferServicesImpl(
     }
 
     @Override
-    public String doExport() throws java.io.IOException {
-        String zipLocation = new StringBuilder()
-            .append(System.getProperty("java.io.tmpdir"))
-            .append(File.separator)
-            .append(FILE_PATH)
-            .append(File.separator)
-            .append(EXPORT_FILE_NAME)
-            .toString();
+    public String doExport(DataTransferExportModel exportModel) throws java.io.IOException {
+        String zipLocation = new StringBuilder().append(System.getProperty("java.io.tmpdir"))
```

Comment:
Consider extracting `System.getProperty("java.io.tmpdir")` to a constant. 
I see it being used more than once on this class.